File: src/plugin/command/administrator/SpawnRunesCommandPlugin.java
package plugin.command.administrator;

import com.rs.game.entity.actor.player.Player;
import com.rs.game.plugin.type.CommandPlugin;
import plugin.command.CommandManifest;

/**
 * @author Tyluur <itstyluur@gmail.com>
 * @since 9/8/2017
 */
@CommandManifest(description = "Spawns all the runes you'll ever need")
public class SpawnRunesCommandPlugin extends CommandPlugin {
	
	@Override
	public void handle(Player player, String[] args, boolean console, boolean clientCommand) {
		final int amount = 10_000;
		for (int i = 554; i <= 566; i++) {
			player.getInventory().addItem(i, amount);
		}
		player.getInventory().addItem(9075, amount);
	}
	
	@Override
	public String[] identifiers() {
		return arguments("runes");
	}
}

File: src/main/java/plugin/command/player/SpawnRunesCommandPlugin.kt
package plugin.command.player

import org.redrune.game.content.plugin.type.CommandPlugin
import org.redrune.game.entity.actor.player.Player
import plugin.command.CommandManifest

/**
 * @author Tyluur <itstyluur@icloud.com>
 * @since 9/8/2017
 */
@CommandManifest(description = "Spawns all the runes you'll ever need")
class SpawnRunesCommandPlugin : CommandPlugin() {
    override fun handle(player: Player, args: Array<String>, console: Boolean, clientCommand: Boolean) {
        val amount = 10000
        for (i in 554..566) {
            player.inventory.addItem(i, amount)
        }
        player.inventory.addItem(9075, amount)
    }

    override fun identifiers(): Array<String> {
        return arguments("runes")
    }
}